extensions/amp-auto-ads/0.1/premiumads-network-config.js

```diff
@@ -45,6 +45,7 @@ export class PremiumadsNetworkConfig {
     return {
       'type': 'doubleclick',
       'data-ad': 'premiumads',
+      'json': data.json || undefined,
```

Comment:
Look like `data.json` would suffice here, unless you explicitly want to convert other falsy value in `data.json` to undefined.

Reply:
At first I thought it was an object, now I changed to `data.json || ''`.

Reply:
Just a reminder that empty string is not a valid json according to spec, and your backend needs to deal with it, but it is up to you on your interface design.